src/controllers/api.controller.js

```diff
@@ -26,8 +26,9 @@ apiCtrl.getAllScreenings = async (req, res) => {
 };
 
 apiCtrl.movieReviews = async (req, res) => {
-  res.json(await loadMovieReviews(req.params.id));
+  res.json(await loadMovieReviews(req.params.id, req.query));
```

Comment:
This seems like a very obvious solution, but what you've done now is couple the way that the external interface (REST API) looks with the way the internal interface (`loadMovieReviews()`) looks. It's not a big problem, but I would have preferred passing in the necessary parameters individually the same way that you do with the movie ID, e.g. `loadMovieReviews(req.params.id, req.query.page)`.